VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -100,7 +116,7 @@ public void onDestroy() {
 
     @SuppressWarnings("unused")
     public void onEventMainThread(DiscussionCommentPostedEvent event) {
-        if (null != event.getParent() && event.getParent().getIdentifier().equalsIgnoreCase(discussionComment.getIdentifier())) {
+        if (null != event.getParent() && event.getParent().getIdentifier().equalsIgnoreCase(discussionResponse.getIdentifier())) {
```

Comment:
Yeah, it doesn't make any sense avoiding the string case for the equality. Fixed it.

> Do we get back inconsistent capitalization

I guess the condition was added to be super-safe if we had inconsistent capitalization in the future.